diff --git a/node_modules/super-simple-web-server/index.js b/node_modules/super-simple-web-server/index.js
index 1eb8065..d21a4db 100644
--- a/node_modules/super-simple-web-server/index.js
+++ b/node_modules/super-simple-web-server/index.js
@@ -58,7 +58,7 @@ app.listen(HTTP_PORT, (err) => {
   if (err) {
     console.log('Http method retrurned an error when trying to start.', err);
   } else {
-    console.log('Listening on: http://' + NODE_HOST + ':' + HTTP_PORT + '');
+    console.log('Listening on: http://' + NODE_HOST + ':' + HTTP_PORT + '?remote');
   }
 });
 
@@ -70,7 +70,7 @@ server.listen(HTTPS_PORT, NODE_HOST, (err) => {
   if (err) {
     console.log('Https method retrurned an error when trying to start.', err);
   } else {
-    console.log('Listening on: https://' + NODE_HOST + ':' + HTTPS_PORT + '');
+    console.log('Listening on: https://' + NODE_HOST + ':' + HTTPS_PORT + '?remote');
   }
 });
 
