diff --git a/node_modules/@strapi/admin/admin/src/StrapiApp.js b/node_modules/@strapi/admin/admin/src/StrapiApp.js
index 7b533ea..780630d 100644
--- a/node_modules/@strapi/admin/admin/src/StrapiApp.js
+++ b/node_modules/@strapi/admin/admin/src/StrapiApp.js
@@ -37,7 +37,7 @@ class StrapiApp {
       notifications: { releases: true },
       themes: { light: lightTheme, dark: darkTheme },
       translations: {},
-      tutorials: true,
+      tutorials: false,
     };
     this.appPlugins = appPlugins || {};
     this.library = library;
diff --git a/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js b/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js
index bf9d59a..cdef19f 100644
--- a/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js
@@ -14,7 +14,7 @@ import { useQueries } from 'react-query';
 import packageJSON from '../../../../package.json';
 import { useConfiguration } from '../../hooks/useConfiguration';
 import { getFullName, hashAdminUserEmail } from '../../utils';
-import { NpsSurvey } from '../NpsSurvey';
+// import { NpsSurvey } from '../NpsSurvey';
 import { PluginsInitializer } from '../PluginsInitializer';
 import RBACProvider from '../RBACProvider';
 
@@ -106,7 +106,7 @@ const AuthenticatedApp = () => {
       userDisplayName={userDisplayName}
     >
       <RBACProvider permissions={permissions} refetchPermissions={refetch}>
-        <NpsSurvey />
+        {/* <NpsSurvey /> */}
         <PluginsInitializer />
       </RBACProvider>
     </AppInfoProvider>
diff --git a/node_modules/@strapi/admin/admin/src/components/NpsSurvey.tsx b/node_modules/@strapi/admin/admin/src/components/NpsSurvey.tsx
index 9ca0ef1..2ee47ce 100644
--- a/node_modules/@strapi/admin/admin/src/components/NpsSurvey.tsx
+++ b/node_modules/@strapi/admin/admin/src/components/NpsSurvey.tsx
@@ -399,7 +399,7 @@ function useNpsSurveySettings() {
   const [npsSurveySettings, setNpsSurveySettings] = usePersistentState<NpsSurveySettings>(
     'STRAPI_NPS_SURVEY_SETTINGS',
     {
-      enabled: true,
+      enabled: false,
       lastResponseDate: null,
       firstDismissalDate: null,
       lastDismissalDate: null,
