diff --git a/node_modules/@strapi/plugin-open-ai/admin/src/components/CMEditView/RightLinksCompo/index.js b/node_modules/@strapi/plugin-open-ai/admin/src/components/CMEditView/RightLinksCompo/index.js
index 182b6b4..607a56e 100644
--- a/node_modules/@strapi/plugin-open-ai/admin/src/components/CMEditView/RightLinksCompo/index.js
+++ b/node_modules/@strapi/plugin-open-ai/admin/src/components/CMEditView/RightLinksCompo/index.js
@@ -224,7 +224,7 @@ const RightLinksCompo = () => {
                           paddingLeft={4}
                           background="neutral0"
                         >
-                          <TextInput
+                          <Textarea
                             placeholder={formatMessage({
                               id: getTrad('Modal.tabs.prompt.placeholder'),
                               defaultMessage:
@@ -234,7 +234,7 @@ const RightLinksCompo = () => {
                             name="content"
                             onChange={(e) => setPrompt(e.target.value)}
                             value={prompt}
-                          />
+                          ></Textarea>
                         </Box>
                         <Box
                           color="neutral800"
@@ -244,6 +244,9 @@ const RightLinksCompo = () => {
                         >
                           <Button
                             paddingTop={4}
+                            paddingRight={3}
+                            paddingBottom={4}
+                            paddingLeft={3}
                             onClick={() => handlePromptSubmit()}
                           >
                             {generateCompletionText}
