diff --git a/node_modules/@strapi/admin/admin/src/StrapiApp.js b/node_modules/@strapi/admin/admin/src/StrapiApp.js
index bbe0ef2..2c60534 100644
--- a/node_modules/@strapi/admin/admin/src/StrapiApp.js
+++ b/node_modules/@strapi/admin/admin/src/StrapiApp.js
@@ -37,7 +37,7 @@ class StrapiApp {
       notifications: { releases: true },
       themes: { light: lightTheme, dark: darkTheme },
       translations: {},
-      tutorials: true,
+      tutorials: false,
     };
     this.appPlugins = appPlugins || {};
     this.library = library;
diff --git a/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js b/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js
index 10c5073..fab4892 100644
--- a/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/AuthenticatedApp/index.js
@@ -14,7 +14,7 @@ import { useQueries } from 'react-query';
 import packageJSON from '../../../../package.json';
 import { useConfigurations } from '../../hooks';
 import { getFullName, hashAdminUserEmail } from '../../utils';
-import NpsSurvey from '../NpsSurvey';
+// import NpsSurvey from '../NpsSurvey';
 import PluginsInitializer from '../PluginsInitializer';
 import RBACProvider from '../RBACProvider';
 
@@ -110,7 +110,7 @@ const AuthenticatedApp = () => {
       userDisplayName={userDisplayName}
     >
       <RBACProvider permissions={permissions} refetchPermissions={refetch}>
-        <NpsSurvey />
+        {/* <NpsSurvey /> */}
         <PluginsInitializer />
       </RBACProvider>
     </AppInfoProvider>
diff --git a/node_modules/@strapi/admin/admin/src/components/NpsSurvey/hooks/useNpsSurveySettings.js b/node_modules/@strapi/admin/admin/src/components/NpsSurvey/hooks/useNpsSurveySettings.js
index eafe5c6..dc7efec 100644
--- a/node_modules/@strapi/admin/admin/src/components/NpsSurvey/hooks/useNpsSurveySettings.js
+++ b/node_modules/@strapi/admin/admin/src/components/NpsSurvey/hooks/useNpsSurveySettings.js
@@ -6,7 +6,7 @@ export function useNpsSurveySettings() {
   const [npsSurveySettings, setNpsSurveySettings] = usePersistentState(
     'STRAPI_NPS_SURVEY_SETTINGS',
     {
-      enabled: true,
+      enabled: false,
       lastResponseDate: null,
       firstDismissalDate: null,
       lastDismissalDate: null,
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
index 84ec0ac..545162b 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/ContentBlocks.js
@@ -1,17 +1,37 @@
 import React from 'react';
 
-import { Flex } from '@strapi/design-system';
-import { ContentBox, useTracking } from '@strapi/helper-plugin';
-import { CodeSquare, FeatherSquare, InformationSquare, PlaySquare } from '@strapi/icons';
-import { useIntl } from 'react-intl';
-import styled from 'styled-components';
-
-import CloudBox from './CloudBox';
+import { Box, Flex, Grid, GridItem } from "@strapi/design-system";
+import { ContentBox, useTracking } from "@strapi/helper-plugin";
+import {
+  FeatherSquare,
+  InformationSquare,
+  ChartBubble,
+} from "@strapi/icons";
+import { useIntl } from "react-intl";
+import styled from "styled-components";
 
 const BlockLink = styled.a`
   text-decoration: none;
 `;
 
+const StyledChartBubble = styled(ChartBubble)`
+  path {
+    fill: #7289da !important;
+  }
+`;
+
+const StyledInformationSquare = styled(InformationSquare)`
+  path {
+    stroke: #7289da !important;
+  }
+`;
+
+const StyledFeatherSquare = styled(FeatherSquare)`
+  path {
+    stroke: #7289da !important;
+  }
+`;
+
 const ContentBlocks = () => {
   const { formatMessage } = useIntl();
   const { trackUsage } = useTracking();
@@ -22,83 +42,76 @@ const ContentBlocks = () => {
 
   return (
     <Flex direction="column" alignItems="stretch" gap={5}>
-      <CloudBox />
-      <BlockLink
-        href="https://strapi.io/resource-center"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonReadTheDocumentationSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'global.documentation',
-            defaultMessage: 'Documentation',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.documentation.content',
-            defaultMessage: 'Discover the essential concepts, guides and instructions.',
-          })}
-          icon={<InformationSquare />}
-          iconBackground="primary100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/starters"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonCodeExampleSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.code',
-            defaultMessage: 'Code example',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.code.content',
-            defaultMessage: 'Learn by using ready-made starters for your projects.',
-          })}
-          icon={<CodeSquare />}
-          iconBackground="warning100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog/categories/tutorials"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonTutorialSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.tutorial',
-            defaultMessage: 'Tutorials',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.tutorial.content',
-            defaultMessage: 'Follow step-by-step instructions to use and customize Strapi.',
-          })}
-          icon={<PlaySquare />}
-          iconBackground="secondary100"
-        />
-      </BlockLink>
-      <BlockLink
-        href="https://strapi.io/blog"
-        target="_blank"
-        rel="noopener noreferrer nofollow"
-        onClick={() => handleClick('didClickonBlogSection')}
-      >
-        <ContentBox
-          title={formatMessage({
-            id: 'app.components.BlockLink.blog',
-            defaultMessage: 'Blog',
-          })}
-          subtitle={formatMessage({
-            id: 'app.components.BlockLink.blog.content',
-            defaultMessage: 'Read the latest news about Strapi and the ecosystem.',
-          })}
-          icon={<FeatherSquare />}
-          iconBackground="alternative100"
-        />
-      </BlockLink>
+      <Grid gap={5}>
+        <GridItem col={4}>
+          <BlockLink
+            href="/admin/content-manager/collectionType/api::post.post"
+          >
+            <ContentBox
+              title={formatMessage({
+                id: "app.components.HomePage.ManagePosts.title",
+                defaultMessage: "Posts",
+              })}
+              subtitle={formatMessage({
+                id: "app.components.HomePage.ManagePosts.subtitle",
+                defaultMessage: "Manage Post Content",
+              })}
+              icon={<StyledFeatherSquare />}
+              iconBackground="primary100"
+            />
+          </BlockLink>
+        </GridItem>
+        <GridItem col={4}>
+          <BlockLink
+            href="/admin/content-manager/singleType/api::home.home"
+          >
+            <ContentBox
+              title={formatMessage({
+                id: "app.components.HomePage.HomeContent.title",
+                defaultMessage: "Edit Homepage",
+              })}
+              subtitle={formatMessage({
+                id: "app.components.HomePage.HomeContent.subtitle",
+                defaultMessage: "Edit Homepage Content",
+              })}
+              icon={<StyledInformationSquare />}
+              iconBackground="primary100"
+            />
+          </BlockLink>
+        </GridItem>
+        <GridItem col={4}>
+          <BlockLink
+            href="https://plausible.io/dgrebb.com"
+            target="_blank"
+            rel="noopener noreferrer nofollow"
+          >
+            <ContentBox
+              title={formatMessage({
+                id: "app.components.HomePage.plausible.title",
+                defaultMessage: "Plausible Analytics",
+              })}
+              subtitle={formatMessage({
+                id: "app.components.HomePage.plausible.subtitle",
+                defaultMessage: "See the traffic",
+              })}
+              icon={<StyledChartBubble />}
+              iconBackground="primary100"
+            />
+          </BlockLink>
+        </GridItem>
+      </Grid>
+      <Box padding={4} shadow="filterShadow">
+        <iframe
+          plausible-embed
+          id="plausible"
+          src="https://plausible.io/share/dgrebb.com?auth=8sm72wfwKETpyquVl_Np_&embed=true&theme=dark"
+          scrolling="no"
+          frameBorder="0"
+          loading="lazy"
+          style={{ width: "1px", minWidth: "100%", height: "2800px" }}
+        ></iframe>
+        <script async src="https://plausible.io/js/embed.host.js"></script>
+      </Box>
     </Flex>
   );
 };
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index 8fb94c5..5ee061b 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -22,7 +22,7 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
             {hasCreatedContentType
               ? formatMessage({
                   id: 'app.components.HomePage.welcome.again',
-                  defaultMessage: 'Welcome 👋',
+                  defaultMessage: "Hello here's a hand 👋",
                 })
               : formatMessage({
                   id: 'app.components.HomePage.welcome',
@@ -34,22 +34,15 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
               ? formatMessage({
                   id: 'app.components.HomePage.welcomeBlock.content.again',
                   defaultMessage:
-                    'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
+                  'Make something',
                 })
               : formatMessage({
                   id: 'app.components.HomePage.welcomeBlock.content',
                   defaultMessage:
-                    'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
+                  'Congrats! You can make stuff.',
                 })}
           </WordWrap>
-          {hasCreatedContentType ? (
-            <Link isExternal href="https://strapi.io/blog">
-              {formatMessage({
-                id: 'app.components.HomePage.button.blog',
-                defaultMessage: 'See more on the blog',
-              })}
-            </Link>
-          ) : (
+          {hasCreatedContentType ? null : (
             <Button size="L" onClick={onCreateCT} endIcon={<ArrowRight />}>
               {formatMessage({
                 id: 'app.components.HomePage.create',
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
index e290ab2..13b36c6 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/SocialLinks.js
@@ -8,17 +8,17 @@ import React from 'react';
 import { Box, Flex, Grid, GridItem, Typography } from '@strapi/design-system';
 import { Link, LinkButton } from '@strapi/design-system/v2';
 import { useAppInfo } from '@strapi/helper-plugin';
-import { Discord, Discourse, ExternalLink, Github, Reddit, Strapi, Twitter } from '@strapi/icons';
+import { ExternalLink, Github, Strapi, Check, Cloud } from '@strapi/icons';
 import { useIntl } from 'react-intl';
 import styled from 'styled-components';
 
-const StyledDiscord = styled(Discord)`
+const StyledGithub = styled(Github)`
   path {
     fill: #7289da !important;
   }
 `;
 
-const StyledReddit = styled(Reddit)`
+const StyledCheck = styled(Check)`
   > path:first-child {
     fill: #ff4500;
   }
@@ -35,82 +35,34 @@ const StyledStrapi = styled(Strapi)`
   }
 `;
 
-const StyledTwitter = styled(Twitter)`
+const StyledCloud = styled(Cloud)`
   path {
     fill: #1da1f2 !important;
   }
 `;
 
-const StyledDiscourse = styled(Discourse)`
-  > path:first-child {
-    fill: #231f20;
-  }
-  > path:nth-child(2) {
-    fill: #fff9ae;
-  }
-  > path:nth-child(3) {
-    fill: #00aeef;
-  }
-  > path:nth-child(4) {
-    fill: #00a94f;
-  }
-  > path:nth-child(5) {
-    fill: #f15d22;
-  }
-  > path:nth-child(6) {
-    fill: #e31b23;
-  }
-`;
-
 const socialLinks = [
   {
-    name: { id: 'app.components.HomePage.community.links.github', defaultMessage: 'Github' },
-    link: 'https://github.com/strapi/strapi/',
-    icon: <Github fill="#7289DA" />,
-    alt: 'github',
+    name: { id: 'app.components.HomePage.github.repo', defaultMessage: 'Github' },
+    link: 'https://github.com/dgrebb/dgrebb.com',
+    icon: <StyledGithub fill="#7289DA" />,
+    alt: 'github repository',
   },
   {
-    name: { id: 'app.components.HomePage.community.links.discord', defaultMessage: 'Discord' },
-    link: 'https://discord.strapi.io/',
-    icon: <StyledDiscord />,
-    alt: 'discord',
+    name: { id: 'app.components.HomePage.github.project', defaultMessage: 'Project' },
+    link: 'https://github.com/users/dgrebb/projects/12/views/1',
+    icon: <StyledCheck />,
+    alt: 'github project',
   },
   {
-    name: { id: 'app.components.HomePage.community.links.reddit', defaultMessage: 'Reddit' },
-    link: 'https://www.reddit.com/r/Strapi/',
-    icon: <StyledReddit />,
-    alt: 'reddit',
-  },
-  {
-    name: { id: 'app.components.HomePage.community.links.twitter', defaultMessage: 'Twitter' },
-    link: 'https://twitter.com/strapijs',
-    icon: <StyledTwitter />,
-    alt: 'twitter',
-  },
-  {
-    name: { id: 'app.components.HomePage.community.links.forum', defaultMessage: 'Forum' },
-    link: 'https://forum.strapi.io',
-    icon: <StyledDiscourse />,
-    alt: 'forum',
-  },
-  {
-    name: { id: 'app.components.HomePage.community.links.blog', defaultMessage: 'Blog' },
-    link: 'https://strapi.io/blog?utm_source=referral&utm_medium=admin&utm_campaign=career%20page',
-    icon: <StyledStrapi />,
-    alt: 'blog',
-  },
-  {
-    name: {
-      id: 'app.components.HomePage.community.links.career',
-      defaultMessage: 'We are hiring!',
-    },
-    link: 'https://strapi.io/careers?utm_source=referral&utm_medium=admin&utm_campaign=blog',
-    icon: <StyledStrapi />,
-    alt: 'career',
+    name: { id: 'app.components.HomePage.aws.link', defaultMessage: 'AWS Console' },
+    link: 'https://dgrebb.signin.aws.amazon.com/console',
+    icon: <StyledCloud />,
+    alt: 'aws console',
   },
 ];
 
-const LinkCustom = styled(LinkButton)`
+const LinkCustom = styled(Link)`
   display: flex;
   align-items: center;
   border: none;
@@ -166,23 +118,10 @@ const SocialLinks = () => {
             <Typography variant="delta" as="h2" id="join-the-community">
               {formatMessage({
                 id: 'app.components.HomePage.community',
-                defaultMessage: 'Join the community',
-              })}
-            </Typography>
-            <Typography textColor="neutral600">
-              {formatMessage({
-                id: 'app.components.HomePage.community.content',
-                defaultMessage:
-                  'Discuss with team members, contributors and developers on different channels',
+                defaultMessage: 'Get to Work',
               })}
             </Typography>
           </Flex>
-          <Link href="https://feedback.strapi.io/" isExternal endIcon={<ExternalLink />}>
-            {formatMessage({
-              id: 'app.components.HomePage.roadmap',
-              defaultMessage: 'See our road map',
-            })}
-          </Link>
         </Flex>
       </Box>
       <GridGap>
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
index a22f746..7457d68 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/index.js
@@ -68,19 +68,16 @@ export const HomePageCE = () => {
         </LogoContainer>
         <Box padding={10}>
           <Grid>
-            <GridItem col={8} s={12}>
+            <GridItem col={12}>
               <HomeHeader
                 onCreateCT={handleClick}
                 hasCreatedContentType={hasAlreadyCreatedContentTypes}
               />
             </GridItem>
           </Grid>
-          <Grid gap={6}>
-            <GridItem col={8} s={12}>
-              {showGuidedTour ? <GuidedTourHomepage /> : <ContentBlocks />}
-            </GridItem>
-            <GridItem col={4} s={12}>
-              <SocialLinks />
+          <Grid>
+            <GridItem col={12}>
+              <ContentBlocks />
             </GridItem>
           </Grid>
         </Box>
diff --git a/node_modules/@strapi/admin/admin/src/translations/en.json b/node_modules/@strapi/admin/admin/src/translations/en.json
index b63fbaf..854369d 100644
--- a/node_modules/@strapi/admin/admin/src/translations/en.json
+++ b/node_modules/@strapi/admin/admin/src/translations/en.json
@@ -432,14 +432,20 @@
   "app.components.GuidedTour.skip": "Skip the tour",
   "app.components.GuidedTour.title": "3 steps to get started",
   "app.components.HomePage.button.blog": "See more on the blog",
-  "app.components.HomePage.community": "Join the community",
+  "app.components.HomePage.community": "Have some links",
   "app.components.HomePage.community.content": "Discuss with team members, contributors and developers on different channels.",
+  "app.components.HomePage.plausible.title": "Analytics Dashboard",
+  "app.components.HomePage.plausible.subtitle": "See recent traffic to dgrebb.com",
+  "app.components.HomePage.HomeContent.title": "Homepage",
+  "app.components.HomePage.HomeContent.subtitle": "Edit Homepage content",
+  "app.components.HomePage.ManagePosts.title": "Posts",
+  "app.components.HomePage.ManagePosts.subtitle": "Manage Post Content",
   "app.components.HomePage.create": "Create your first Content type",
   "app.components.HomePage.roadmap": "See our roadmap",
-  "app.components.HomePage.welcome": "Welcome on board 👋",
-  "app.components.HomePage.welcome.again": "Welcome 👋",
+  "app.components.HomePage.welcome": "Welcome on board 🧐",
+  "app.components.HomePage.welcome.again": "Make Something 👌",
   "app.components.HomePage.welcomeBlock.content": "Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!",
-  "app.components.HomePage.welcomeBlock.content.again": "We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.",
+  "app.components.HomePage.welcomeBlock.content.again": "Write something beautiful.",
   "app.components.HomePage.welcomeBlock.content.issues": "issues.",
   "app.components.HomePage.welcomeBlock.content.raise": " or raise ",
   "app.components.ImgPreview.hint": "Drag & drop your file into this area or {browse} for a file to upload",
