diff --git a/src/exports/vite/index.js b/src/exports/vite/index.js
index e80fb786aca949d57431406ab6a16394ed82a90a..6ec870b698711cacfb908ef52fb9f30b1a7eb995 100644
--- a/src/exports/vite/index.js
+++ b/src/exports/vite/index.js
@@ -635,8 +635,8 @@ async function kit({ svelte_config }) {
 							input,
 							output: {
 								format: 'esm',
-								entryFileNames: ssr ? '[name].js' : `${prefix}/[name].[hash].${ext}`,
-								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[name].[hash].${ext}`,
+								entryFileNames: ssr ? '[name].js' : `${prefix}/[hash].${ext}`,
+								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[hash].${ext}`,
 								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
 								hoistTransitiveImports: false,
 								sourcemapIgnoreList
@@ -650,8 +650,8 @@ async function kit({ svelte_config }) {
 					worker: {
 						rollupOptions: {
 							output: {
-								entryFileNames: `${prefix}/workers/[name]-[hash].js`,
-								chunkFileNames: `${prefix}/workers/chunks/[name]-[hash].js`,
+								entryFileNames: `${prefix}/workers/[hash].js`,
+								chunkFileNames: `${prefix}/workers/chunks/[hash].js`,
 								assetFileNames: `${prefix}/workers/assets/[name]-[hash][extname]`,
 								hoistTransitiveImports: false
 							}
