diff --git a/src/exports/vite/index.js b/src/exports/vite/index.js
index 98e487c4fae7f6dd485dcdbd87eb6fdcab7140f3..e12ba0a5b9a5ed5fc5bcf9acaab7c801c3daaba7 100644
--- a/src/exports/vite/index.js
+++ b/src/exports/vite/index.js
@@ -612,8 +612,8 @@ async function kit({ svelte_config }) {
 							input,
 							output: {
 								format: 'esm',
-								entryFileNames: ssr ? '[name].js' : `${prefix}/[name].[hash].${ext}`,
-								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[name].[hash].${ext}`,
+								entryFileNames: ssr ? '[name].js' : `${prefix}/[hash].${ext}`,
+								chunkFileNames: ssr ? 'chunks/js' : `${prefix}/chunks/[hash].${ext}`,
 								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
 								hoistTransitiveImports: false,
 								sourcemapIgnoreList
@@ -627,8 +627,8 @@ async function kit({ svelte_config }) {
 					worker: {
 						rollupOptions: {
 							output: {
-								entryFileNames: `${prefix}/workers/[name]-[hash].js`,
-								chunkFileNames: `${prefix}/workers/chunks/[name]-[hash].js`,
+								entryFileNames: `${prefix}/workers/[hash].js`,
+								chunkFileNames: `${prefix}/workers/chunks/[hash].js`,
 								assetFileNames: `${prefix}/workers/assets/[name]-[hash][extname]`,
 								hoistTransitiveImports: false
 							}
