diff --git a/src/exports/vite/index.js b/src/exports/vite/index.js
index 94d511ab86e7dfb86c95c53faff302b4e243914d..791868a4abca35f9aba1f1bc50b3f02462a8bb49 100644
--- a/src/exports/vite/index.js
+++ b/src/exports/vite/index.js
@@ -636,8 +636,8 @@ async function kit({ svelte_config }) {
 							input,
 							output: {
 								format: 'esm',
-								entryFileNames: ssr ? '[name].js' : `${prefix}/[name].[hash].${ext}`,
-								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[name].[hash].${ext}`,
+								entryFileNames: ssr ? '[name].js' : `${prefix}/[hash].${ext}`,
+								chunkFileNames: ssr ? 'chunks/[name].js' : `${prefix}/chunks/[hash].${ext}`,
 								assetFileNames: `${prefix}/assets/[name].[hash][extname]`,
 								hoistTransitiveImports: false,
 								sourcemapIgnoreList
@@ -651,8 +651,8 @@ async function kit({ svelte_config }) {
 					worker: {
 						rollupOptions: {
 							output: {
-								entryFileNames: `${prefix}/workers/[name]-[hash].js`,
-								chunkFileNames: `${prefix}/workers/chunks/[name]-[hash].js`,
+								entryFileNames: `${prefix}/workers/[hash].js`,
+								chunkFileNames: `${prefix}/workers/chunks/[hash].js`,
 								assetFileNames: `${prefix}/workers/assets/[name]-[hash][extname]`,
 								hoistTransitiveImports: false
 							}
